apps/curation_front_end/src/store/settings.store.ts

```diff
@@ -10,6 +10,10 @@ import { persist } from "zustand/middleware";
 
 type State = {
     treeWidth: number;
+    user: {
+        username: string;
+        role: string;
+    } | null;
```

Comment:
What's the thinking behind storing user in Settings store instead of using the `useUser` hook?

Reply:
I explained it in the PR comment, but now that you said that, I realized I should have just passed it on to the components as a prop. I'll change that now